packages/entities/entities-plugins/src/components/free-form/shared/utils.ts

```diff
@@ -3,3 +3,30 @@ export function toSelectItems<T extends string>(
 ): { value: T; label: T }[] {
   return items.map((item) => ({ value: item, label: item }))
 }
+
+export const path = {
+  arraySymbol: '*',
+  rootSymbol: '$',
+  separator: '.',
```

Comment:
Maybe we shouldn't allow to customize these symbols. And I think it's better to export the following methods just as plain functions like `resolvePath`, `resolveRootPath`, etc.